linux_os/guide/system/network/network-firewalld/ruleset_modifications/configure_firewalld_rate_limiting/bash/shared.sh

```diff
@@ -1,7 +1,7 @@
 # platform = multi_platform_all
 
 common_firewalld_ratelimit_args=(--direct --add-rule ipv4 filter INPUT_direct 0 -p tcp -m limit --limit 25/minute --limit-burst 100  -j INPUT_ZONES)
-if {{{ in_chrooted_environment }}}; then
+if {{{ in_chrooted_environment }}} || {{{ bash_bootc_build() }}}; then
```

Comment:
I wouldn't do this change because this rule is used only in the OL 7 default profile, so I will have difficulty to test it.

Reply:
Ok, let me revert it.